app/redux/thunks/CartThunks.js

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+
+import { setCart, addItemToCart, _removeItemFromCart } from "../actions";
+// temporary fetchCart thunk
+export function fetchCart() {
+  return function thunk(dispatch) {
+    return axios
+      .get(`/api/cart`)
+      .then(res => res.data)
+      .then(products => dispatch(setCart(products)))
+      .catch(e => console.error("Error fetching Cart", e));
+  };
+}
+/*
+export function fetchCart() {
+  return function thunk(dispatch) {
+    return axios
+      .get(`/api/cart/${userId}`)
+      .then(res => res.data)
+      .then(products => dispatch(setCart(products)))
+      .catch(e => console.error("Error fetching Cart", e));
+  };
+}
+
+export function addItemToCart() {
+  return function thunk(dispatch) {
+    return axios
+      .post(`/api/cart/add/${userId}`, product)
+      .then(res => res.data)
+      .then(product => dispatch(_addItemToCart(product)))
+      .catch(e => console.error("Error adding Cart item", e));
+  };
+}
+
+export function removeItemFromCart(product) {
```

Comment:
pass a pass a userId and a product. api call will be /api/user/${ userId }/cart/${ productId }